zio-http-example/src/main/scala/example/StaticServer.scala

```diff
@@ -3,47 +3,53 @@ package example
 import zio.ZIOAppDefault
 
 import zio.http._
+import zio.http.codec.PathCodec.trailing
 import zio.http.html._
 
 object StaticServer extends ZIOAppDefault {
 
   // A simple app to serve static resource files from a local directory.
-  val app = Http
-    .collectHandler[Request] { case Method.GET -> "" /: "static" /: path =>
+  val app = Routes(
+    Method.GET / "static" / trailing -> handler {
+      val extractPath    = Handler.param[(Path, Request)](_._1)
+      val extractRequest = Handler.param[(Path, Request)](_._2)
+
       for {
-        file <- Handler.getResourceAsFile(path.encode)
+        path <- extractPath
+        file <- extractRequest >>> Handler.getResourceAsFile(path.encode)
```

Comment:
I think `extractRequest` is not needed for neither this nor on line 22, because both the `getResourceAsFile` and `template` handlers are independent of their input

Reply:
The `Handler.notFound` is where the problem is. It requires a request.

Reply:
I was able to delete the other one on line 19.